app/controllers/publish/courses/funding_type_controller.rb

```diff
@@ -1,6 +1,6 @@
 module Publish
   module Courses
-    class FeeOrSalaryController < PublishController
+    class FundingTypeController < PublishController
```

Comment:
I think this might clash with https://github.com/DFE-Digital/publish-teacher-training/pull/3002/files#diff-00c01d3e1da4c3c7b634a815925bd22c2257c268edf082e4e3ba9e18dbc18ee0R3 ? cc @defong